packages/docs/content/components/checkbox.mdx

```diff
@@ -89,18 +89,46 @@ This component also makes use of form field styles, which can be customized by t
 - You may need to add the className `.ds-u-margin--0` to your child element(s) to avoid extra top margin
 - If you opt for smaller radio buttons or checkboxes, add className `.ds-c-choice__checkedChild--small` to your checked child container
 
-### Accessibility
+### Accessibility testing
 
-- Surround a related set of choices with a `<fieldset>`. The `<legend>` provides context for the grouping. Do not use `fieldset` and `legend` for a single checkbox.
-- Some screen readers read the `legend` text for each `fieldset`, so it should be brief and descriptive.
+#### General observations
+
+- No other interactive elements (such as links or buttons) should be included inside the label or hint text.
```

Comment:
> or hint text

I think that was something we solved with the new changes around separating label, hint, and error